<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Resume</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cerulean.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="resume.html">Resume</a>
</li>
<li>
  <a href="mailto:xinyi_shang@outlook.com">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/xinyi-anna-shang/">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/XinyiShang/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Resume</h1>

</div>


<div id="xinyi-shang" class="section level1">
<h1>Xinyi Shang</h1>
<p><a
href="https://www.linkedin.com/in/xinyi-shang-27a47026b/">LinkedIn</a> |
<a href="https://github.com/XinyiShang">GitHub</a> | <a
href="file/Shang_Xinyi_Oct28.pdf">Resume</a></p>
<hr />
<div id="education" class="section level2">
<h2>Education</h2>
<div id="columbia-university-mailman-school-of-public-health"
class="section level3">
<h3>Columbia University, Mailman School of Public Health</h3>
<div id="new-york-ny-092023---exp-052025" class="section level4">
<h4>New York, NY | 09/2023 - Exp 05/2025</h4>
<ul>
<li>Master of Science in <strong>Biostatistics</strong></li>
<li>Theory and Methods Track | GPA: 4.00/4.00</li>
</ul>
</div>
</div>
<div id="brandeis-university" class="section level3">
<h3>Brandeis University</h3>
<div id="waltham-ma-092019---062023" class="section level4">
<h4>Waltham, MA | 09/2019 - 06/2023</h4>
<ul>
<li>B.S. in <strong>Biology</strong> with Honors; Minor in
<strong>Computer Science</strong></li>
<li>GPA 3.83/4.00 | <em>summa cum laude</em>; Dean’s List 2020 -
2023</li>
</ul>
<hr />
</div>
</div>
</div>
<div id="academic-experience" class="section level2">
<h2>Academic Experience</h2>
<div id="identifying-cell-type-specific-spatially-variable-genes"
class="section level3">
<h3>Identifying Cell-type-specific Spatially Variable Genes</h3>
<div id="supervisor-prof.-wenpin-hou-prof.-zhicheng-ji-062024-present"
class="section level4">
<h4>Supervisor: Prof. Wenpin Hou, Prof. Zhicheng Ji |
06/2024-Present</h4>
<ul>
<li>This project integrates genomic and spatial data to identify
spatially variable genes within specific cell types in spatial
transcriptomes, aiding in the understanding of regional biological
processes and functions.</li>
<li>Assisted in annotating cell types using 10x Visium HD datasets from
various species and tissues, including mouse brain, human pancreas,
human lung cancer, mouse embryo, and human colorectal cancer. Explored
genetic and cell type databases such as Panglao DB and CellMarker 2.0 to
gain insights into cell type-specific gene expression.</li>
<li>Evaluated algorithms and analyzed ontology terms to enhance the
understanding of gene expression patterns and their spatial
distribution, contributing to the analysis of gene function and spatial
heterogeneity in organs and tissues.</li>
</ul>
</div>
</div>
<div id="geneturing-tests-gpt-models-in-genomics"
class="section level3">
<h3>GeneTuring tests GPT models in genomics</h3>
<div id="supervisor-prof.-wenpin-hou-prof.-zhicheng-ji-082024-present"
class="section level4">
<h4>Supervisor: Prof. Wenpin Hou, Prof. Zhicheng Ji |
08/2024-Present</h4>
<ul>
<li>This project focused on evaluating advanced GPT models in their
ability in answering genomic questions.</li>
<li>Conducted thorough assessments of evaluating advanced GPT models
(GPT-3.5, GPT-4, GPT-4o, Gemini Advance) across 1,200 genomics-related
questions, benchmarking their performance on tasks such as gene
extraction, SNP location, and gene-disease association, protein coding,
gene ontology, gene alias association, etc.</li>
<li>Analyzed model accuracy by focusing on the models’ ability to
recognize task limitations and handle unknown queries.</li>
</ul>
</div>
</div>
<div
id="thalamic-nuclei-derived-radiomics-volumetric-trajectories-in-ms"
class="section level3">
<h3>Thalamic Nuclei Derived Radiomics &amp; Volumetric Trajectories in
MS</h3>
<div id="supervisor-dr.-korhan-buyukturkoglu-052024-062024"
class="section level4">
<h4>Supervisor Dr. Korhan Buyukturkoglu | 05/2024-06/2024</h4>
<ul>
<li>Analyzing radiomic and volumetric data to uncover the relationship
between thalamic nuclei changes in Multiple Sclerosis (MS) patients and
cognitive impacts measured by the Symbol Digit Modalities Test
(SDMT).</li>
<li>Processed and analyzed data from 126 MS patients, focusing on 8
bilateral thalamic nuclei, the whole thalamus, and over 1,500 radiomic
features.</li>
<li>Identified 2 distinct MS subtypes based on volumetric and 2 subtypes
using radiomic data.</li>
</ul>
</div>
</div>
<div
id="dynamical-synaptic-strengths-and-its-effect-on-the-discrimination-of-sequential-stimuli"
class="section level3">
<h3>Dynamical Synaptic Strengths and its Effect on the Discrimination of
Sequential Stimuli</h3>
<div id="senior-thesis" class="section level4">
<h4>Senior Thesis</h4>
</div>
<div id="supervisor-prof.-paul-miller-062022---052023"
class="section level4">
<h4>Supervisor: Prof. Paul Miller | 06/2022 - 05/2023</h4>
<ul>
<li>Conducted a study on the effect of short-term synaptic plasticity on
the discrimination of different stimuli sequences within a randomly
connected attractor network.</li>
<li>Utilized a computational neuroscience model for analysis.</li>
<li>Employed clustering and confusion matrix techniques to investigate
the network’s ability to discriminate with various parameter sets.</li>
<li>Results demonstrated that the presence of facilitation increased
network stability, while depression enhanced sensitivity to different
signals. The network with both facilitation and depression exhibited the
most complex behavior.</li>
<li><a
href="https://github.com/XinyiShang/DynamicalSynapticDynamics_SeqDiscrimination">Link
to this Project</a></li>
</ul>
<hr />
</div>
</div>
</div>
<div id="manuscript-poster-abstract" class="section level2">
<h2>Manuscript &amp; Poster Abstract</h2>
<p>Zhuang, H., <strong>Shang, X.</strong>, Hou, W., &amp; Ji, Z. (in
progress). <em>Identifying Cell-Type-Specific Spatially Variable Genes
with ctSVG2.</em></p>
<p>Buyukturkoglu, K., Davis, L., Wen, S., <strong>Shang, X.</strong>,
Zhang, W., Comandate-Lou, N., Blackwelder, J., Shende, V. K., Ozcelik,
S., Boulanger, A., Riley, C., Stern, Y., &amp; De Jager, P. (2024).
<em>Thalamic Nuclei Derived Radiomics and Volumetric Trajectories in
Multiple Sclerosis and Their Associations with Symbol Digit Modalities
Test.</em> Presented at the 40th Congress of ECTRIMS, Copenhagen,
Denmark.</p>
<hr />
</div>
<div id="awards" class="section level2">
<h2>Awards</h2>
<div id="grand-prize-winner-hacking-health-hackathon-2024"
class="section level3">
<h3><strong>Grand Prize Winner</strong>, Hacking Health Hackathon
2024</h3>
<div id="columbia-university-022024" class="section level4">
<h4>Columbia University | 02/2024</h4>
<ul>
<li><strong>Project</strong>: Developed a <strong>Sepsis Early-Stage
Prediction Model</strong> using Random Forest, achieving ~98% accuracy
by analyzing patient vital signs, biomarkers, and demographic
data.<br />
</li>
<li><strong>Deployment</strong>: Built a Flask-based website for
real-time model deployment.<br />
</li>
<li><strong>Media Coverage</strong>:
<ul>
<li><a
href="https://www.patientsafetytech.com/past-competitions/hackinghealth">Hacking
Health (Patient Safety Technology Challenge)</a>,</li>
<li><a
href="https://www.columbiaspectator.com/arts-and-culture/2024/02/07/students-discuss-healthcare-technologies-at-hacking-health-hackathon/">Columbia
Spectator</a>,</li>
<li><a
href="https://ppahs.org/2024/03/hacking-health-and-how-you-can-improve-sepsis-care/">Physician-Patient
Alliance for Health &amp; Safety</a>,</li>
<li><a
href="https://jhf.org/news/three-patient-safety-technology-challenge-competitions-announce-winning-innovations-1-2">Jewish
Healthcare Foundation</a>,</li>
<li>and <em>Biostatistics Department Weekly Updates</em>.</li>
</ul></li>
</ul>
</div>
</div>
<div id="silver-medal-hms---harmful-brain-activity-classification"
class="section level3">
<h3><strong>Silver Medal</strong>, HMS - Harmful Brain Activity
Classification</h3>
<div id="kaggle-competition-022024042024" class="section level4">
<h4>Kaggle Competition | 02/2024–04/2024</h4>
<ul>
<li>Collaborated on a <strong>deep learning model</strong> for EEG
pattern classification using EEG data and spectrograms.<br />
</li>
<li>Managed <strong>106,800 observations</strong>, applying analytical
methods to boost model accuracy.<br />
</li>
<li>Designed an ensemble model (ResNet-1D, GRU, EfficientNet-B1) with
<strong>ROI-focused data augmentation</strong>, improving detection of
complex brain activities.</li>
</ul>
</div>
</div>
<div id="semi-finalist-womens-health-tech-challenge"
class="section level3">
<h3><strong>Semi-finalist</strong>, Women’s Health Tech Challenge</h3>
<div id="hitlab-082024092024" class="section level4">
<h4>HITLAB| 08/2024–09/2024</h4>
<ul>
<li><p>Participated as part of Team <a
href="https://ai4purpose.com/">AI4Purpose Inc</a>. Contributed to
<strong>Antisepsis for Infants and Moms (AIM)</strong>, focused on
creating digital health tools to combat maternal sepsis.</p></li>
<li><p>Developed a health tracking device based on the Antisepsis
framework to assess sepsis risk using patient biomedical markers.
Enhanced the machine learning model to more accurately target maternal
and neonatal sepsis.</p></li>
<li><p>Featured in [NYC ASA: Chapter News(<a
href="https://www.nycasa.org/news.html"
class="uri">https://www.nycasa.org/news.html</a>).</p></li>
</ul>
<hr />
</div>
</div>
</div>
<div id="teaching-experience" class="section level2">
<h2>Teaching Experience</h2>
<p><strong>Teaching Assistant</strong>, Applied Regression II, Columbia
University<br />
<em>09/2024–Present</em></p>
<p><strong>Teaching Assistant</strong>, Statistical Computing with SAS,
Columbia University<br />
<em>09/2024–Present</em></p>
<p><strong>Facilitator</strong>, Summer Health Professions Education
Program (SHPEP), Columbia University<br />
<em>06/2024–07/2024</em><br />
- Assisted in teaching <strong>Organic Chemistry, Physics, and Narrative
Medicine</strong> while facilitating the <strong>Physical Therapy
track</strong>.<br />
- Managed course logistics, including attendance tracking, grading
assignments, and completing clerical tasks.<br />
- Led a small group of ~10 students and organized two social activities
to foster engagement and community.</p>
<hr />
</div>
<div id="intership" class="section level2">
<h2>Intership</h2>
<div id="rd-intern" class="section level3">
<h3>R&amp;D Intern</h3>
<div id="tiangen-biotechbeijing-co.-ltd-022021-052021"
class="section level4">
<h4>Tiangen Biotech(Beijing) Co., Ltd | 02/2021-05/2021</h4>
<ul>
<li>Purified genomes from blood and plants</li>
<li>Evaluated the effective of genome purification kits</li>
<li>Helped test the degradation of DNA by different RNAse
preservative</li>
</ul>
<hr />
</div>
</div>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><strong>Programming Languages</strong>: Python, R, MATLAB, Java,
SAS, SQL<br />
</li>
<li><strong>Machine Learning &amp; Deep Learning</strong>: Regression
Analysis, Classification, Clustering, Neural Networks, Computer
Vision<br />
</li>
<li><strong>Biostatistics &amp; Statistical Modeling</strong>: Survival
Analysis, Longitudinal Analysis, Clinical Trial Design, Linear Mixed
Models<br />
</li>
<li><strong>Data Analysis &amp; Tools</strong>: Data Mining, Exploratory
Data Analysis (EDA), Simulation, Shiny Apps, API Integration<br />
</li>
<li><strong>Software &amp; Platforms</strong>: Microsoft Office (Word,
Excel, PowerPoint), Anaconda, SPSS, RStudio, MySQL</li>
</ul>
<hr />
</div>
<div id="related-projects" class="section level2">
<h2>Related Projects</h2>
<div id="dia-mostic" class="section level3">
<h3>Dia-mostic</h3>
<div
id="columbia-healthcare-ventures-fast-pitch-competition-092024-present"
class="section level4">
<h4>Columbia Healthcare Ventures Fast-Pitch Competition |
09/2024-Present</h4>
<ul>
<li>Developed <strong>Dia-mostic</strong>, an AI-powered solution aimed
at improving the diagnosis and management of <strong>gestational
diabetes</strong> for healthier maternal and neonatal outcomes.<br />
</li>
<li>Created a predictive model using machine learning algorithms to
enhance early detection of gestational diabetes and improve patient
care.<br />
</li>
<li>Built an interactive mobile app (<a
href="https://app.uizard.io/p/1a43c69e">prototype available here</a>) to
provide healthcare providers and patients with real-time insights,
personalized recommendations, and tracking tools.</li>
</ul>
</div>
</div>
<div id="alpha-skin-ai-enabled-solution-for-posture-monitoring"
class="section level3">
<h3>Alpha Skin: AI-Enabled Solution for Posture Monitoring</h3>
<div id="rutgers-health-hackathon-2024-102024" class="section level4">
<h4>Rutgers Health Hackathon 2024 | 10/2024</h4>
<ul>
<li>Participated in the Rutgers Hackathon as part of Team Alpha Skin,
developing an AI-driven skin sensor solution designed for real-time
monitoring of spinal motion and posture correction.<br />
</li>
<li>Technical Approach:
<ul>
<li>Cloud-based model: Extracted key posture features from cleaned data
and updated the system using a population-level databank.<br />
</li>
<li>Personalized Monitoring: Initialized individual models based on
user-specific posture patterns, enabling real-time, tailored
feedback.<br />
</li>
<li>Built an interactive mobile app (<a
href="https://app.uizard.io/p/ed93eb2c/preview">prototype available
here</a>).</li>
</ul></li>
</ul>
</div>
</div>
<div id="breast-cancer-survival-prediction" class="section level3">
<h3>Breast Cancer Survival Prediction</h3>
<div id="course-project-112023-122023" class="section level4">
<h4>Course Project | 11/2023-12/2023</h4>
<ul>
<li>Conducted statistical analysis on a <strong>prospective cohort study
dataset</strong> to identify key predictors of breast cancer risk.<br />
</li>
<li>Developed a <strong>Multiple Logistic Regression model</strong>
incorporating variables such as age, race, tumor characteristics, and
hormone status, <strong>achieving 85.6% accuracy</strong> in predicting
survival outcomes.<br />
</li>
<li>Applied advanced statistical techniques for data cleaning,
processing, and analysis, including regression analysis and hypothesis
testing to uncover significant patterns in demographic and disease
characteristics.<br />
</li>
<li><a
href="https://github.com/XinyiShang/p8130_final_project/blob/main/final_report.pdf">Link
to this Project</a></li>
</ul>
</div>
</div>
<div id="analysis-of-nypd-complaint-data" class="section level3">
<h3>Analysis of NYPD Complaint Data</h3>
<div id="course-project-102023-122023" class="section level4">
<h4>Course Project | 10/2023-12/2023</h4>
<ul>
<li>Performed a comprehensive analysis of NYPD complaint data to uncover
trends and patterns in criminal complaints across New York City.<br />
</li>
<li>Cleaned, processed, and analyzed large datasets using advanced
statistical techniques, including regression analysis, to extract
insights from crime data.<br />
</li>
<li>Developed interactive visualizations using a <strong>Shiny
app</strong> and created a <strong>GitHub website</strong> with
<strong>R Markdown</strong> to present findings.</li>
<li><a
href="https://xinyishang.github.io/p8105_final_project_NYPD.github.io/">Link
to this Project</a></li>
</ul>
</div>
</div>
<div
id="improve-the-network-performance-based-on-pruning-filters-for-efficient-convnets"
class="section level3">
<h3>Improve the network performance based on “Pruning Filters for
Efficient ConvNets”</h3>
<div id="course-project-supervisor-prof.-hongfu-liu-042023-052023"
class="section level4">
<h4>Course Project | Supervisor: Prof. Hongfu Liu | 04/2023-05/2023</h4>
<ul>
<li>Implemented the network from the paper by Hao Li et al., focusing on
pruning of filters with the smallest weight to improve computational
efficiency and reduce model size</li>
<li>Explored the transition from VGG16 to VGG19 architectures, and
introduced Dropout regulation to VGG16 model, resulting in nominal
improvements</li>
<li>Employed quantization methods to decrease the size of the VGG16
model while maintaining accuracy levels, but still have difficulties in
assessing the pruned network’s accuracy</li>
<li><a
href="https://github.com/XinyiShang/COSI159_PruningImprovement">Link to
this Project</a></li>
</ul>
<hr />
</div>
</div>
</div>
<div id="certifications" class="section level2">
<h2>Certifications</h2>
<ul>
<li><strong>SAS Certified Specialist</strong>: Base Programming Using
SAS 9.4<br />
</li>
<li><strong>SAS Certified Professional</strong>: Advanced Programming
Using SAS 9.4</li>
</ul>
<hr />
</div>
<div id="extracurricular-activities" class="section level2">
<h2>Extracurricular Activities</h2>
<ul>
<li><strong>Volunteer</strong>, Language Empowering Action Project<br />
<em>09/2022–05/2023</em>
<ul>
<li>Tutored adults in <strong>English language skills</strong>, helping
them improve communication and literacy.</li>
</ul></li>
<li><strong>Volunteer</strong>, Leland Home (C2E Program)<br />
<em>09/2019–12/2019</em>
<ul>
<li>Provided <strong>companionship and support</strong> to elderly
residents, fostering social interaction and well-being.</li>
</ul></li>
</ul>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
